csr/csr.go

```diff
@@ -74,8 +74,10 @@ func VerifyCSR(csr *x509.CertificateRequest, maxNames int, keyPolicy *goodkey.Ke
 	if maxNames > 0 && len(csr.DNSNames) > maxNames {
 		return fmt.Errorf("CSR contains more than %d DNS names", maxNames)
 	}
+	names := make(map[string]bool, len(csr.DNSNames))
```

Comment:
Sorry, I missed this in the last round. I think the `names` map you create & populate in `VerifyCSR` isn't needed anymore and should be deleted.